app/controllers/orga/seasons_controller.rb

```diff
@@ -45,23 +45,13 @@ def destroy
   end
 
   # # switch_phase: enables developers to easily switch between time dependent settings in views
-  # by opening and closing the corresponding links in the nav bar
+  # by showing the corresponding links in the nav bar
   def switch_phase
     return if Rails.env.production?
-      case params[:option]
-      when 'Proposals'
-        Season::PhaseSwitcher.fake_proposals_phase
-      when 'Application'
-        Season::PhaseSwitcher.fake_application_phase
-      when 'CodingSummer'
-        Season::PhaseSwitcher.fake_coding_phase
-      when 'RealTime'
-        Season::PhaseSwitcher.back_to_reality
-      end
-    redirect_to orga_seasons_path, notice: "We time travelled into the #{params[:option]} phase"
+    Season::PhaseSwitcher.destined(params[:phase])
```

Comment:
While I was changing this, I remembered [this conversation](https://github.com/rails-girls-summer-of-code/rgsoc-teams/pull/585#discussion_r96146778) about the DoS risk with converting user input strings to symbols (and the magic . Aren't we reintroducing the risk if we convert upfront?